reflex/app.py

```diff
@@ -1138,6 +1139,9 @@ async def process(
                 # assignment will recurse into substates and force recalculation of
                 # dependent ComputedVar (dynamic route variables)
                 state.router_data = router_data
+            if state.router:
+                state.router.update(router_data)
+            else:
```

Comment:
if we update the `router` outside of the `if state.router_data != router_data:` block, then that means _every_ event will trigger a change and delta to the `router` Var (and any dependent computed vars); this could increase network traffic pretty significantly since the serialized `router` object is relatively large.

I wonder if it makes sense to keep the connection `status` and `last_event` fields on a separate object that is not sent to the frontend. I don't really see either of these being super useful for frontend rendering, they're mainly for checking from event handlers from my perspective.

Reply:
Thanks for your review and suggestions. Indeed, we do not really need `status` and `last_event` in the frontend. I adjusted the session state to be a separate backend var.
If someone really needs `status` or `last_event` in the frontend, it's still possible to wrap them in a `@rx.var` as seen in the updated PR description.